request.js

```diff
@@ -89,8 +89,15 @@ SuiteRequest.prototype = {
 
   _getLogParameters: function(options) {
     return _.pick(options, ['method', 'host', 'url']);
-  }
+  },
 
+  _getPayload: function(data) {
+    if (this._options.getHeader('content-type') !== 'application/json') {
```

Comment:
- It won't work with this header: Content-Type: application/json; charset=utf-8
- It won't work with non lowercased header value Application/JSON (is this a real example?)

I think you should check for partial match with lowercased values